website/snippets/_legacy-sl-callout.md

```diff
@@ -0,0 +1,11 @@
+:::tip Upgrade to access the new dbt Semantic Layer
+
+The dbt Semantic Layer has undergone a [significant revamp](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/), improving governance, introducing a new API, and making it more efficient to define/query metrics. This means the legacy Semantic Layer, available in dbt v1.5 or lower, is no longer supported and won't receive any code fixes.
+
+**Who does this affect?** Anyone who uses the legacy Semantic Layer. The new Semantic Layer is available to [Team or Enterprise](https://www.getdbt.com/pricing/) multi-tenant dbt Cloud plans [hosted in North America](/docs/cloud/about-cloud/regions-ip-addresses). You must be on dbt v1.6 or higher to access it. Users on dbt Cloud Developer plans or dbt Core users can use MetricFlow to only define and test metrics locally.
```

Comment:
These summaries are really good and nicely laid out! So easy to read. I was thinking it's a little long, but I can't find anything to cut. 🤷🏻‍♀️